llvm/test/DebugInfo/Generic/artificial-static-member.ll

```diff
@@ -1,3 +1,4 @@
+; REQUIRES: x86_64-linux
 ; RUN: llc -O0 -filetype=obj < %s |   \
```

Comment:
The requirement will never be satisfied; the test will never run. You need a regular expression of the form `target={{x86_64-.*-linux}}` here.

But, it's probably better to remove the `target triple` clause from the IR, and add `-triple=%itanium_target_triple` to the `llc` command.